src/Kiota.Builder/Refiners/CommonLanguageRefiner.cs

```diff
@@ -182,21 +186,21 @@ protected static void AddGetterAndSetterMethods(CodeElement current, HashSet<Cod
                 ReturnType = (CodeTypeBase)currentProperty.Type.Clone(),
                 Documentation = new()
                 {
-                    Description = $"Gets the {currentProperty.WireName} property value. {currentProperty.Documentation.Description}",
+                    Description = $"Gets the {currentProperty.Name} property value. {currentProperty.Documentation.Description}",
```

Comment:
I'm curious about this change: isn't this going to result in the comment mentioning the sanitized name instead of the serialized property name?

Reply:
I sort of remember an issue that I can't find anymore for some reason where the ask was to get the serialization name of the property so it's easier to mentally map since we do all sorts of normalization, escaping etc...

Reply:
right, we could improve javadoc comments further down the line, but let's consider this a separate thing.
let's stick with wirename for now so we don't introduce a regression (again I can't find the original customer ask anymore, might have been an internal email)

Reply:
👍 reverted.